src/views/catalog/wizard/components/WizardFooter.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { useURLParams } from '@kubevirt-utils/hooks/useURLParams';
+import { getResourceUrl } from '@kubevirt-utils/resource';
+import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  Button,
+  Checkbox,
+  Split,
+  SplitItem,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+
+import { clearSessionStorageVM, useWizardVMContext } from '../../utils/WizardVMContext';
+
+export const WizardFooter: React.FC<{ namespace: string }> = ({ namespace }) => {
+  const history = useHistory();
+  const { t } = useKubevirtTranslation();
+  const { params } = useURLParams();
+  const templateName = params.get('name');
+  const templateNamespace = params.get('namespace');
+
+  const { vm } = useWizardVMContext();
+  const [loaded, setLoaded] = React.useState(true);
+  const [error, setError] = React.useState<any>();
+  const [startVM, setStartVM] = React.useState(true);
+
+  const onCreate = () => {
+    setLoaded(false);
+    setError(undefined);
+
+    vm.metadata.namespace = namespace;
+    if (startVM) {
+      vm.spec.running = true;
+    }
+
+    k8sCreate<V1VirtualMachine>({
+      model: VirtualMachineModel,
+      data: vm,
+    })
+      .then(() => {
+        clearSessionStorageVM();
+        history.push(getResourceUrl(VirtualMachineModel, vm));
```

Comment:
@yaacov what do we want to do upon vm creation?
right now we navigate to the vm's details page

Reply:
sounds right to me